regimes/mx/invoice_validator.go

```diff
@@ -10,6 +11,9 @@ import (
 	"github.com/invopop/validation"
 )
 
+// SAT item identity codes (ClaveProdServ) regular expression.
+var itemIdentityCodeRegexp = regexp.MustCompile(`^\d{8}$`)
```

Comment:
Is it worth allowing 6 digits and assuming the last two are `00`? Probably not... just curious :-)

Reply:
We can do it! Would you do it as a kind of "normalisation" of the code and add the missing 00 when on document calculation?

Reply:
That could indeed make sense! Might be hard to get the normalization to the item identity level though. I think just validating either 6 or 8 would be sufficient.

Reply:
Cool! Since that padding of zeroes should happen either here or in `gobl.cfdi` anyway (i.e., the SAT doesn't accept 6-digit codes), I gave it a try to do it here and I think the resulting code is good enough (particularly after the refactor that #168 enabled). See e825e46, and let me know if you have any feedback.

Reply:
I wasn't sure if that would work! Nice one!